app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -22,7 +24,26 @@ def self.event_monitor_class
 
   def create_project(project)
     connect.create_project_request(project)
-  rescue KubeException => e
```

Comment:
@simon3z I was asked to catch this exception here and re-raise a corresponding miq-exception as part of the container template life-cycle so that the automation side can handle it accordingly. 
cc @bzwei